Client/Teztez/Core/Language.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Language.swift
+//  Teztez
+//
+//  Created by Almas Zainoldin on 4/18/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+enum Language: String, CaseIterable {
+    case eng
+}
+
+extension Language {
```

Comment:
I think we can do better and use some framework or better solution to deal with Localization. 

Our app is going to have quite nice UX and changing the language should also be nice and be updated "on the fly" (realtime). The same way Telegram does it.

But I believe that our current implementation is not capable of doing that. (You implement localization as a separate feature)

Reply:
I think we should create new issue and do all localization logic in it.